device/core/test/device_method/_device_method_response_test.js

```diff
@@ -114,7 +114,7 @@ describe('DeviceMethodResponse', function() {
       // test & assert
       res.send(200);
       assert.throws(function() {
-        res.send(202);
+        res.send(202, undefined, function(err) {});
```

Comment:
>res.send(202, undefined, function(err) {}) [](start = 8, length = 42)

Sorry, could I get a short explanation on why the call changed?

Reply:
Previous tests weren't passing callbacks when they didn't care about the result value. This caused all the calls to return promises after my changes.

Reply:
Done.

---
In reply to: [222781839](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222781839) [](ancestors = 222781839)